docs/content/developer/tutorials/sponsored-txs.mdx

```diff
@@ -0,0 +1,365 @@
+1- Check if the user has already requested funds. If they have, it will return a conflict status.
+2- Add the user's address to the set of addresses that have requested funds.
+3- Build a signed and funded transaction using the `sign_and_fund_transaction` util function, which we will implement next.
+4- Increment the total sponsored fees by the gas price. This is used for tracking by the sponsor.
+5- Respond with the transaction bytes and the sponsor's signature.
+
+Next, we will implement the `sign_and_fund_transaction` util function, which will build the transaction, sign it, and return the signed transaction.
+under the `utils.rs` file, add the following code:
+
+```rust reference
+https://github.com/iota-community/sponsored-transactions-demo/blob/main/backend/src/utils.rs#L119-L208
+```
+
+This function will:
+1- Build a programmable transaction that calls the `free_trial` function of the `sponsored_transactions_packages` package.
+2- Get the gas coin of the sponsor.
+3- Create a transaction data object with the programmable transaction, gas budget, gas price, and sponsor address.
+4- Sign the transaction by the sponsor.
+5- Return the signed transaction.
+
+
+Finally, we can now run the server and call the `/sign_and_fund_transaction` route to sponsor the free trial. The server will return the transaction bytes and the sponsor's signature, which the user can use to send the transaction to the IOTA network.
+
+In a terminal, run the server with the following command:
+
+```shell
+cargo run
+```
+
+In another terminal, send a POST request to the /sign_and_fund_transaction route with the following cURL call (replace the sender address and content type with your own):
+```shell
+curl -X POST \
+    -H "Content-Type: application/json" \
+    -d '{
+        "sender": "0xc7d158a9b05c6dfd07c233649c9e0f78320d066e5ac0e8f5101de500ad9e84e8",
+        "content_type": "Music"        
+    }' \
+    http://localhost:3001/sign_and_fund_transaction
+```
+
+The server will return the transaction bytes and the sponsor's signature, which the user can use to send the transaction to the IOTA network.
+
+```json
+{
+"message":"Transaction signed and funded successfully",
+"bytes":"AAACAAYFTXVzaWMBAbBSNubKBn4/oRS6sVWPNeRAl+AHiqaBdh+qYiIIWKF2DhQAAAAAAAABAQAgaekcgzM1C99rvSmRJmrTOZJ1fbevSCka21jnpbDhqh9zcG9uc29yZWRfdHJhbnNhY3Rpb25zX3BhY2thZ2VzCmZyZWVfdHJpYWwAAgEAAAEBAMfRWKmwXG39B8IzZJyeD3gyDQZuWsDo9RAd5QCtnoToATnqmj+YicH8PXnWPZeuKTp0hZ0M4Lh2wJAhFHHjcGe7ERQAAAAAAAAgAakPl91BjzezX1DLa+BnLWkJ5k0vnTX6/Qg/iJw11gW/KTztJZMRjNIx8QfzQbsa2ds5zQSXv/KdNVcwz04rwugDAAAAAAAAQEtMAAAAAAAA",
+"sigs":["ACCPERsBa2Wt7TFO1p7KDbDI5epikVJEmwvE3yLKv3nuMvgMVFZbh9EJ0kQjLAlHGfUaYMEvrZuNMSuR7zOqrA/p1vxEC3McO9IvqEI8BXaHBgCUFx3DXTO+QWmQKwj2uA=="]
+}
+```
+
+
+For the full code of the Gas Station server, you can check the [Gas Station server implementation](https://github.com/iota-community/sponsored-transactions-demo/blob/main/backend/src/main.rs)
+
+
+
+### Third Component: Transaction Builder
```

Comment:
Seeing that the tutorial will have muitliple parts, wdyt about splitting the docs by components?

Reply:
That would be nicer yes, will separate them 👍